src/reporter.rs

```diff
@@ -7,31 +7,36 @@ use crate::analyzer;
 
 pub fn run(results: &HashMap<String, analyzer::Report>, output_file: &str) -> std::io::Result<()> {
     // TODO: Add metrics measures %
-    let mut html = String::new();
-    html.push_str("<html><head><title>React Hooks Analyzer Report</title>");
-    html.push_str("<style>");
-    html.push_str("body { font-family: Arial, sans-serif; margin: 20px; }");
-    html.push_str("h1 { color: #333; }");
-    html.push_str("table { width: 100%; border-collapse: collapse; margin-top: 20px; }");
-    html.push_str("th, td { border: 1px solid #ccc; padding: 8px; text-align: left; }");
-    html.push_str("th { background-color: #f4f4f4; }");
-    html.push_str("tr:nth-child(even) { background-color: #f9f9f9; }");
-    html.push_str("</style>");
-    html.push_str("</head><body>");
-    html.push_str("<h1>Custom React Hooks Analyzer Report</h1>");
-    html.push_str(&format!("<p>Files visited: {}</p>", results.len()));
-
-    html.push_str("<table>");
-    html.push_str(
-        "<tr>
+    let mut html = format!(
+    r#"<!DOCTYPE html>
+    <html>
+        <head>
+            <title>React Hooks Manuel Report</title>
```

Comment:
[nit]: the page title should be: **React Hooks Analyzer Report**
```suggestion
            <title>React Hooks Analyzer Reportt</title>
```

Reply:
The title is already fixed.